pings.yaml

```diff
@@ -9,6 +9,19 @@
 ---
 $schema: moz://mozilla.org/schemas/glean/pings/2-0-0
 
+study-enrollment:
+  description: |
+    This ping is sent when the user has consented to the study.
+  include_client_id: false
+  send_if_empty: true
+  bugs:
+    - https://github.com/mozilla-rally/rally-study-01/issues/114
```

Comment:
If you re-use the data-review as recommended below, please keep both the old issue/bug and the new one (it's fine for this to be a list!)